docs/style.css

```diff
@@ -0,0 +1,78 @@
+h3 {
```

Comment:
The features section content does not touches the ends of the browser. This is good as any modern web page has this. To accomplish this, you have applied padding to individual elements or subsections of the feature section. You can improve this by wrapping all the feature section in container div and apply the padding to that. Then if you want to move any other element/section inside, then you can apply padding to that.